### ISSUE TITLE
Notifications not working on Android and IOS

### ISSUE THREAD

**Issue by ShadowDragonX** [lotusgate/otraku#63]
First thanks for the app. Its really awesome. The only down site is that notifications of new episode releases/airing are not working.

On Android:
When i first installed the app the notifications worked for like 2 or 3 days but after that i never received any notifications at all. When i check the feed i do see that there are new airing notifications avaiable. Im using Android 9

On IOS: 
i installed it through altstore and i never received notifications from this app at all
Im using IOS 14.8

If u could look into it i'd appreciate it. 
Thanks in advance.

**Comment by lotusgate** [comment]
IOS notifications are not yet implemented.
For android: can you navigate to Settings->Apps->Otraku and check the battery settings? On some devices, battery optimisers/limiters prevent the app from fetching notifications in the background, so this could be the issue. On another note, can you confirm that the app is otherwise working correctly for you? AL had issues recently, so I want to make sure the problem is not related.

**Comment by ShadowDragonX** [comment]
> IOS notifications are not yet implemented. 

- is it on your to-do list?

> For android: can you navigate to Settings->Apps->Otraku and check the battery settings? On some devices, battery optimisers/limiters prevent the app from fetching notifications in the background, so this could be the issue. 

- Optimizing battery usage and allow background activity both are enabled 
- i disabled them once but it didnt fix it

> On another note, can you confirm that the app is otherwise working correctly for you? AL had issues recently, so I want to make sure the problem is not related.

- everything else is working good so far

**Comment by lotusgate** [comment]
No, background activity _should_ be allowed. But the battery optimiser being on may prevent the app from fetching notifications.

**Comment by YouSummoner** [comment]
This isn't a problem in the app, Anilist itself has currently stopped sending notifications for the time being, also all the notification settings from the Anilist website have also disappeared...

**Comment by lotusgate** [comment]
@YouSummoner notification settings haven't disappeared. There's a problem with the api for setting updates and notification settings can literally be erased by a faulty query from an app/extension/script or something else.
If you ask AL support, they may give you a script that could restore you notification settings.

**Comment by YouSummoner** [comment]
> @YouSummoner notification settings haven't disappeared. There's a problem with the api for setting updates and notification settings can literally be erased by a faulty query from an app/extension/script or something else.
> If you ask AL support, they may give you a script that could restore you notification settings.

You mean restore on the website or otraku?

**Comment by lotusgate** [comment]
> You mean restore on the website or otraku?

On the website, but that should restore it on Otraku for you too. I don't know what's the particular problem with ShadowDragonX

**Comment by YouSummoner** [comment]
> > You mean restore on the website or otraku?
> 
> On the website, but that should restore it on Otraku for you too. I don't know what's the particular problem with ShadowDragonX

I see, I'll try E-mailing AL support and update on the matter you then.

**Comment by ShadowDragonX** [comment]
> No, background activity _should_ be allowed. But the battery optimiser being on may prevent the app from fetching notifications.

even if i turn it off i do not receive a notification

**Comment by lotusgate** [comment]
Not sure what could be causing this then. What brand is your phone?

**Comment by ShadowDragonX** [comment]
> Not sure what could be causing this then. What brand is your phone?

Samsung Note 8 with andoird 9
